src/LaunchDarkly.Client/LdStoreEventsFactory.cs

```diff
@@ -0,0 +1,11 @@
+﻿

```

Comment:
In order to wrap ```IStoreEvents```, and pass it to ```LdClient``` at construction, I need to be able to construct the built-in ```EventProcessor```.

I felt it was better to expose this as a factory, to allow you to more easily change implementations over time.  For example, if you add a ```StreamingEventProcessor``` down the road, you can hide those details in the factory.